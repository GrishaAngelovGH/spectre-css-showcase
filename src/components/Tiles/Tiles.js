import Avatar from '../Avatar'

const Tiles = () => (
  <div className='columns'>
    <div className='column col-4 col-mx-auto bg-gray' style={{ marginTop: 50, marginBottom: 50 }}>
      <div className='tile'>
        <div className='tile-icon'>
          <figure className='avatar avatar-lg'><Avatar initials='JS' /></figure>
        </div>
        <div className='tile-content'>
          <p className='tile-title'>The Avengers</p>
          <p className='tile-subtitle'>Earth's Mightiest Heroes joined forces to take on threats that were too big for any one hero to tackle...</p>
        </div>
        <div className='tile-action'>
          <button className='btn btn-primary'>Join</button>
          <button className='btn'>Contact</button>
        </div>
      </div>
    </div>
  </div>
)

export default Tiles