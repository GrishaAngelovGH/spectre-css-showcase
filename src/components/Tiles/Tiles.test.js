import { render } from '@testing-library/react'

import Tiles from './Tiles'

test('should render Tiles component', () => {
  const view = render(<Tiles />)

  expect(view).toMatchSnapshot()
})