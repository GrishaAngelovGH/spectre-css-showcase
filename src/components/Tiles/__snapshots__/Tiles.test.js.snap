// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Tiles component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="columns"
      >
        <div
          class="column col-4 col-mx-auto bg-gray"
          style="margin-top: 50px; margin-bottom: 50px;"
        >
          <div
            class="tile"
          >
            <div
              class="tile-icon"
            >
              <figure
                class="avatar avatar-lg"
              >
                <figure
                  class="avatar avatar-sm mx-2"
                  data-initial="JS"
                  style="background: rgb(87, 85, 217);"
                />
              </figure>
            </div>
            <div
              class="tile-content"
            >
              <p
                class="tile-title"
              >
                The Avengers
              </p>
              <p
                class="tile-subtitle"
              >
                Earth's Mightiest Heroes joined forces to take on threats that were too big for any one hero to tackle...
              </p>
            </div>
            <div
              class="tile-action"
            >
              <button
                class="btn btn-primary"
              >
                Join
              </button>
              <button
                class="btn"
              >
                Contact
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="columns"
    >
      <div
        class="column col-4 col-mx-auto bg-gray"
        style="margin-top: 50px; margin-bottom: 50px;"
      >
        <div
          class="tile"
        >
          <div
            class="tile-icon"
          >
            <figure
              class="avatar avatar-lg"
            >
              <figure
                class="avatar avatar-sm mx-2"
                data-initial="JS"
                style="background: rgb(87, 85, 217);"
              />
            </figure>
          </div>
          <div
            class="tile-content"
          >
            <p
              class="tile-title"
            >
              The Avengers
            </p>
            <p
              class="tile-subtitle"
            >
              Earth's Mightiest Heroes joined forces to take on threats that were too big for any one hero to tackle...
            </p>
          </div>
          <div
            class="tile-action"
          >
            <button
              class="btn btn-primary"
            >
              Join
            </button>
            <button
              class="btn"
            >
              Contact
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
